kolibri/plugins/learn/assets/src/views/ProgressBar.vue

```diff
@@ -62,12 +80,58 @@
       },
     },
     computed: {
+      isQuiz() {
+        return lodashGet(this.contentNode, ['options', 'modality'], false) === 'QUIZ';
+      },
       progress() {
         return this.contentNodeProgressMap[this.contentNode && this.contentNode.content_id] || 0;
       },
       completed() {
         return this.progress >= 1;
       },
+      quizProgress() {
+        return (
+          this.contentNodeProgressMetaDataMap[this.contentNode && this.contentNode.content_id] || {}
+        );
+      },
+      remainingQuestions() {
+        if (this.isQuiz && this.quizProgress.total_questions) {
+          return this.quizProgress.total_questions - this.quizProgress.num_question_answered;
+        }
+        return 0;
+      },
+      score() {
+        if (this.isQuiz && this.quizProgress.total_questions) {
+          return (
+            this.quizProgress.num_question_answered_correctly / this.quizProgress.total_questions
+          );
+        }
+        return 0;
+      },
+      inProgressLabel() {
+        if (this.isQuiz) {
+          return this.$tr('questionsLeft', { questionsLeft: this.remainingQuestions });
+        }
+        return '';
+      },
+      completedLabel() {
```

Comment:
Likewise this could be `quizCompletedLabel`. Otherwise it's slightly confusing why there is `completedLabel` in the template above and then `coreString('completedLabel')` below it.